# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gst-plugins-base/hotfix-gl.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gst-plugins-base-1.18.5/gst-libs/gst/gl/meson.build.vanilla	2021-12-27 13:38:23.917073536 +0100
+++ gst-plugins-base-1.18.5/gst-libs/gst/gl/meson.build	2021-12-27 13:38:39.161072928 +0100
@@ -293,16 +293,7 @@
 #endif
 '''
 
-# convoluted way of getting at the subproject taking into account the wrap-mode
-# so we don't download a subproject unless allowed
-gl_header_dep = dependency('', fallback : ['gl-headers', 'gl_headers_dummy_dep'],
-    required : false)
-if gl_header_dep.type_name() == 'internal'
-  # this will only contain the includes of headers that are not found
-  compat_includes = subproject('gl-headers').get_variable('compatibility_includes')
-else
   compat_includes = []
-endif
 
 # Desktop OpenGL checks
 gl_dep = unneeded_dep
